storage/engine/engine_test.go

```diff
@@ -717,10 +718,14 @@ func TestApproximateSize(t *testing.T) {
 		}
 
 		insertKeysAndValues(keys, values, engine, t)
-		if rocksdb, ok := engine.(*RocksDB); ok {
-			err := rocksdb.Flush()
+
+		type flusher interface {
```

Comment:
Can we not just add Flush to the Engine interface? Do we have any non-flushing engines left?